pkg/apis/v1/labels.go

```diff
@@ -54,6 +54,7 @@ func init() {
 }
 
 var (
+	Group                  = "karpenter.k8s.aws"
```

Comment:
Why are we defining this here? I was looking for us to do something like this with the provider, since we have the `apis.Group` https://github.com/kubernetes-sigs/karpenter/blob/a122e25df3decf77daf0930d5fb11f7c91c6b7ca/pkg/apis/apis.go#L26